executor/show.go

```diff
@@ -614,7 +615,8 @@ func (e *ShowExec) fetchShowCreateTable() error {
 			if mysql.HasNotNullFlag(col.Flag) {
 				buf.WriteString(" NOT NULL")
 			}
-			if !mysql.HasNoDefaultValueFlag(col.Flag) {
+			// don't show default value for generated columns
```

Comment:
I think we'd better to explain why "not show default value for generated columns".
For example, "Default value are not shown for generated columns in mysql".
The current comment, I think, is the same as the code(a little bit redundant)

Reply:
Reasonable ~